packages/edit-post/src/test/editor.native.js

```diff
@@ -1,13 +1,16 @@
 /**
  * External dependencies
  */
-import renderer from 'react-test-renderer';
 import RNReactNativeGutenbergBridge from 'react-native-gutenberg-bridge';
-
+import { mount } from 'enzyme';
 /**
  * WordPress dependencies
  */
 import { registerCoreBlocks } from '@wordpress/block-library';
+// Force register 'core/editor' store.
+import { store } from '@wordpress/editor'; // eslint-disable-line no-unused-vars
+
+jest.mock( '../components/layout', () => () => 'Layout' );
```

Comment:
seems the tests pass without this line. Do you remember why you added it?

Reply:
I do get some errors commenting that line 🤔
Erros like:
`
 Error: Uncaught [Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
`

Reply:
Oh right, I was on the wrong repo. 👍